### DISCUSSION TITLE
Move away from deprecated iTextSharp PDF library

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [clawsoftware/clawPDF#74] [Ideas]
Hey, first thanks for your project.
I noticed you are using iTextSharp PDF library which is deprecated, and only bug fixes are issued since 2016. As all of their development effort has been moved to iText7. So instead of using a long abandoned project which lacks newer features, if you are interested, you can look into few well maintained C# pdf libraries I found.
[PdfSharp](https://github.com/empira/PDFsharp)
[PDFjet](https://github.com/edragoev1/pdfjet) - [Website](https://pdfjet.com/)
[QPdfNet](https://github.com/Sicos1977/QPdfNet) - C# wrapper for qpdf
[PdfPig](https://github.com/UglyToad/PdfPig) - C# port of PDFBox
[Haru PDF Library](https://github.com/libharu/libharu) - C# wrapper included
[QuestPDF](https://github.com/QuestPDF/QuestPDF)

**Comment by clawsoftware** [comment]
Hi,

Thank you for your request. Upgrading to iText 7 is on my to-do list. However, I will need to rewrite some code due to significant breaking changes.

**Comment by clawsoftware** [comment]
Was implemented in version 0.9.1

**Comment by bobastler** [comment]
Ganz vielen lieben Dank :)
